Main.java

```diff
@@ -0,0 +1,50 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+    static String scanLine() {
+        Scanner scan = new Scanner(System.in);
+        return scan.nextLine();
+    }
+    public static void main(String[] args) {
+        int start, end, length, var, aux;

```

Comment:
var is a keyword in Java 10+.
In mod ideal, fiecare declaratie de variabila e pe propria ei linie. Aici merge si asa, dar ca sa tii minte.